vm/class.go

```diff
@@ -36,14 +36,14 @@ type RClass struct {
 var externalClasses = map[string][]ClassLoader{}
 var externalClassLock sync.Mutex
 
-// RegisterExternalClass will add the given class to the global registery of available classes
+// RegisterExternalClass will add the given class to the global registration of available classes
```

Comment:
Think `registry` will be a better option?

Reply:
Sorry, I should fix this as suggested.